content/blog/2021/06/2021-06-17-libera-chat.adoc

```diff
@@ -0,0 +1,20 @@
+---
+:layout: post
+:title: "Jenkins IRC on Libera Chat"
+:tags:
+- community
+:author: markewaite
+opengraph:
+  image: /images/post-images/2021-06-libera-chat/jenkins-irc-on-libera-chat-opengraph.png
+---
+
+image:/images/post-images/2021-06-libera-chat/jenkins-irc-on-libera-chat-opengraph.png[Jenkins IRC now on Libera Chat, role=center, link="https://groups.google.com/g/jenkinsci-dev/c/pEcun5ZSfgo/m/ztRg03VNAgAJ"]
+
+The link:/chat/#internet-relay-chat-irc[Jenkins IRC chat channels] have moved to Libera chat.
+Many Jenkins developers have been dropped without warning from the previous Jenkins IRC chat system on Freenode.
+Rather than remain in a system that was increasingly unfriendly to our developers and administrators, the Jenkins project has moved to link:https://libera.chat/[Libera Chat IRC].
```

Comment:
```suggestion
Rather than remain in a system that was increasingly unfriendly to our developers and administrators, the Jenkins community has moved to link:https://libera.chat/[Libera Chat IRC].
```